builder/Makefile.tracee

```diff
@@ -0,0 +1,217 @@
+.PHONY: all
+all:
+
+#
+# make
+#
+
+MAKEFLAGS += --no-print-directory
+
+#
+# tools
+#
+
+CMD_DOCKER ?= docker
+
+.ONESHELL:
+.check_%:
+#
+	@command -v $* >/dev/null
+	if [ $$? -ne 0 ]; then
+		echo "missing required tool $*"
+		exit 1
+	else
+		touch $@ # avoid target rebuilds due to inexistent file
+	fi
+
+#
+# usage
+#
+
+.PHONY: help
+help:
+	@echo ""
+	@echo "To generate tracee container(s):"
+	@echo ""
+	@echo "    $$ make -f builder/Makefile.tracee build-alpine-tracee-core"
```

Comment:
Do we plan to have build-ubuntu-tracee-core one day? If not maybe we could just say build-tracee-core?

Reply:
I thought about this as well. I'm glad you asked. I thought we could, indeed, have the ubuntu flavored containers but not to publish them. This gives us more flexibility to test things (and allow our users to generate ubuntu based containers if they want). But, it was just an idea.

Reply:
Go it. However, I'd separate Dockerfile and Makefiles that are absolutely necessary to perform tracee release from build artifacts that are used more for compatibility testing. WDYT? I'm afraid that a "release manager" might be overwhelmed one day with the number of build config files.

Reply:
Anyway, I'll approve for now.

Reply:
We could move them to the root directory (the release related stuff). I'm just afraid that having a Dockerfile in the root directory is "dirty". Having a "release/" directory sounds good to you ? Or is it "too much" ?

Reply:
My original idea was to have *anything* related to building stuff under "builder/" directory (Makefiles, Docker files, Vagrant files, etc). But we can deviate from that original idea (no hard feelings for it).

Reply:
We can refine it in subsequent PRs. Another option is to add subfolder under builder, e.g. builder/release, builder/compatibility, builder/tests ...